### PR TITLE
feat: Add TRPC

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

**Review by jonator — COMMENTED** [review]
What else is missing? LGTM, please mark ready for review when ready.

**Review by jonator — APPROVED** [review]

File: packages/web/server/api/trpc.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/**
+ * YOU PROBABLY DON'T NEED TO EDIT THIS FILE, UNLESS:
```
Comment (MaxMillington) [inline packages/web/server/api/trpc.ts:2]: good notes

File: packages/web/utils/trpc.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        },
+      ],
+    };
+  },
+  /**
+   * Whether tRPC should await queries when server rendering pages.
+   *
+   * @see https://trpc.io/docs/nextjs#ssr-boolean-default-false
+   */
+  ssr: false,
```
Comment (MaxMillington) [inline packages/web/utils/trpc.ts:93]: Why is ssr false?

**Review by MaxMillington — APPROVED** [review]
Just had one small question

File: packages/web/utils/trpc.ts (hunk 2 above)
Comment (JoseRFelix) [reply packages/web/utils/trpc.ts]: This was the default. Enabled it on my latest commit

**Pull request merged**